BuyPolish/Pola/UI/ProductSearch/Keyboard/BPKeyboardViewController.m

```diff
@@ -0,0 +1,19 @@
+#import "BPKeyboardViewController.h"
+
+@interface BPKeyboardViewController ()
+
+@end
+
+@implementation BPKeyboardViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    self.view.backgroundColor = [UIColor colorWithWhite:0.0 alpha:0.7];
+}
+
+- (BOOL)isPresented {
+    return self.view.superview != nil;
+}
```

Comment:
I think it is not needed. `BPScanCodeView` should now if it have keyboard view presented or not